### DISCUSSION TITLE
Creating new users on NOS 2

### DISCUSSION THREAD

**Discussion by ghost** [kej715/DtCyber#131] [Show and tell]
I was following the steps on the [DtCyber Guide.pdf](https://github.com/kej715/DtCyber/blob/main/doc/DtCyber%20Guide.pdf) for account creation, and I found some issues with the _Typical User Permissions (Figure 2-13)_  on page 2-38, when comparing with the _60459840E NOS 2 Administration Handbook_. I'm assuming the reason is because the Guide refers to NOS 1 and I'm using NOS 2.

- _NF=7_ (number of local files) complains as _OBSOLETE DIRECTIVE IGNORED_. I cannot find it in the Handbook, so I ignored it.
- _PL=77777777B_ (peak connect limit) complains  as _ERROR IN IDENTIFIER_. I cannot find it in the Handbook, so I ignored it.
- _PO=0_ (Robust between carriage returns) complains as _ERROR IN IDENTIFIER_. I cannot find it in the Handbook, so I ignored it.
- _OFF=CSOJ_ and _OFF=CTPC_ are used to toggle the access control word (AW), but it complains as _ERROR IN IDENTIFIER_. I cannot find it in the Handbook. Instead, you can toggle by doing _AW=CSOJ_ and _AW=CTPC_, assuming you previously did _AW=ALL_

Unless I'm mistaken with these assumptions and somebody can correct me, I hope this helps others.

**Comment by wschaub** [comment]
The following NOS2 manuals have good information on MODVAL: 
https://bitsavers.org/pdf/cdc/cyber/nos2/60459840E_NOS_Version_2_Administration_Handbook_Jun92.pdf

In addition you can also do something like this from IAF to get a printout of documentation on the current version of MODVAL on NOS 2.8.7:
log in as install and then:
```
/attach,opl=opl871
/modify.
? *edit modval
? 
 MODIFICATION COMPLETE.
/docment,l=list.
 DOCUMENTATION COMPLETE.
/route,list,dc=pr.
 ROUTE COMPLETE.  JSN IS AADL.
/
````

**Reply by ghost** [reply]
Indeed, that's exactly the Handbook I was using.

Thanks a lot on the tip about how to get the printout of documentation. I'll definitely try tonight.

**Comment by wschaub** [comment]
It looks like NF= was used previously for a limit on the amount of local files the user could have in a job or something like that, but now it's ignored. 

Some of those others might have even been local site mods done by MECC if it was the NOS1.4 system the dtcyber guide was originally written for. 

Looking at OPL552 it does look like ON = and OFF = are part of a custom modset called SETVAL. 
PL is also a custom option to handle peak hour limits. 

I can't find option PO but it's probably the same deal.

**Reply by ghost** [reply]
OK, then we can assume that ignoring NF, PL and PO, and using AW instead of OFF is a safe bet, right?

**Reply by wschaub** [reply]
That's what I'm thinking. if it's not in the NOS2 documentation you can ignore it as specific to either NOS1 or the MECC ready to run system the dtcyber manual was written for (which was a customized version of NOS 1.4)
